From 5e1648d5fe62246bed58218741256d7aee8b11b6 Mon Sep 17 00:00:00 2001
From: Rob Pike <r@golang.org>
Date: Tue, 8 Sep 2015 10:58:21 -0700
Subject: [PATCH 19/63] [release-branch.go1.5] doc: mention that go install
 removes binaries built by go build

Fixes #12288.

For inclusion in the 1.5.1 release.

Change-Id: I9354b7eaa76000498465c4a5cbab7246de9ecb7c
Reviewed-on: https://go-review.googlesource.com/14382
Reviewed-by: Ian Lance Taylor <iant@golang.org>
Reviewed-on: https://go-review.googlesource.com/14394
---
 doc/go1.5.html | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/doc/go1.5.html b/doc/go1.5.html
index 4e5832d..693a185 100644
--- a/doc/go1.5.html
+++ b/doc/go1.5.html
@@ -505,6 +505,13 @@ now require SWIG 3.0.6 or later.
 </li>
 
 <li>
+The <code>install</code> subcommand now removes the
+binary created by the <code>build</code> subcommand
+in the source directory, if present,
+to avoid problems having two binaries present in the tree.
+</li>
+
+<li>
 The <code>std</code> (standard library) wildcard package name
 now excludes commands.
 A new <code>cmd</code> wildcard covers the commands.
-- 
2.6.1

